src/analysis/rust_type.rs

```diff
@@ -373,7 +381,57 @@ impl<'env> RustTypeBuilder<'env> {
+                                    || matches!(inner_type, Basic(Pointer))
+                                    || (matches!(inner_type, Record(_))
+                                        && self
+                                            .env
+                                            .config
+                                            .objects
+                                            .get(&inner_tid.full_name(&self.env.library))
+                                            .map(|o| !o.boxed_inline)
+                                            .unwrap_or(false));
+                                let is_string = matches!(inner_type, Basic(Utf8));
+                                match *type_ {
+                                    CArray(_) if is_ptr => {
+                                        format!("{}<{typ}>", use_glib_type(self.env, "PtrSlice"))
+                                    }
+                                    CArray(_)
+                                        if is_string
+                                            && matches!(
+                                                self.c_type.as_str(),
+                                                "char**" | "gchar**"
+                                            ) =>
+                                    {
+                                        use_glib_type(self.env, "StrV")
+                                    }
+                                    CArray(_)
+                                        if !matches!(
+                                            inner_type,
+                                            Basic(Utf8 | Filename | OsString | Boolean)
+                                        ) =>
+                                    {
+                                        format!("{}<{typ}>", use_glib_type(self.env, "Slice"))
+                                    }
+                                    List(_) if is_ptr => {
+                                        format!("{}<{typ}>", use_glib_type(self.env, "List"))
+                                    }
+                                    List(_) if is_string => format!(
+                                        "{}<{}>",
+                                        use_glib_type(self.env, "List"),
+                                        use_glib_type(self.env, "GStringPtr")
+                                    ),
+                                    SList(_) if is_ptr => {
+                                        format!("{}<{typ}>", use_glib_type(self.env, "SList"))
+                                    }
+                                    SList(_) if is_string => format!(
+                                        "{}<{}>",
+                                        use_glib_type(self.env, "SList"),
+                                        use_glib_type(self.env, "GStringPtr")
+                                    ),
+                                    _ => format!("Vec<{typ}>"),
```

Comment:
Which cases are left here? Only `PtrArray`?

Reply:
I suppose `GArray` as well? But we need wrappers with type parameters

Reply:
Can you add a comment for listing the remaining cases?